lib/generators/cable_ready/templates/config/initializers/cable_ready.rb

```diff
@@ -19,4 +19,9 @@
   # Change the default Active Job queue used for broadcast_later and broadcast_later_to
   #
   # config.broadcast_job_queue = :default
+
+  # Specify a default debounce time for CableReady::Updatable callbacks
+  # Doing so is a best practice to avoid heavy ActionCable traffic
+  #
+  # config.updatable_debounce_time = 0.1.seconds
```

Comment:
@julianrubisch I think we should also add this to:

https://github.com/stimulusreflex/stimulus_reflex/blob/main/lib/generators/stimulus_reflex/templates/config/initializers/cable_ready.rb

Reply:
👍🏻